src/opm/output/eclipse/Summary.cpp

```diff
@@ -2774,7 +2774,7 @@ eval(const EclipseState&                es,
         evalPtr->update(sim_step, duration, input, simRes, st);
     }
 
-    for (auto& [_, evalPtr] : this->extra_parameters) {
+    for (auto& [[[maybe_unused]]_, evalPtr] : this->extra_parameters) {
```

Comment:
Seems the `[[maybe_unused]]` must be in front of the full declaration: `([[maybe_unused]] auto&[_, evalPtr])` - but then the attribute will apply to *both* `_` and `evalPtr`. An alternative based on `(void)_` here: #2111